view/widgets/acquisition_widgets/volume_model.py

```diff
@@ -286,8 +286,8 @@ def move_fov_query(self, new_fov_pos):
         msgBox = QMessageBox()
         msgBox.setIcon(QMessageBox.Question)
         msgBox.setText(f"Do you want to move the field of view from "
-                       f"{[round(x, 2) * t for x, t in zip(self.fov_position, self.polarity)]} to "
-                       f"{[round(x, 2) for x in new_fov_pos]}?")
+                       f"{[round(x, 2) * t for x, t in zip(self.fov_position, self.polarity)]} [mm] to "
+                       f"{[round(x, 2) for x in new_fov_pos]} [mm]?")
```

Comment:
Can you add a unit variable as an init argument with a default of mm, pass unit into the volume model in the volume widget and then use unit instead of hardcoding it?